app/forms/idv/api_image_upload_form.rb

```diff
@@ -51,44 +129,33 @@ def selfie
       as_readable(:selfie)
     end
 
-    def document_capture_session_uuid
-      params[:document_capture_session_uuid]
-    end
-
     def document_capture_session
       @document_capture_session ||= DocumentCaptureSession.find_by(
         uuid: document_capture_session_uuid,
       )
     end
 
-    def self.human_attribute_name(attr, options = {})
-      # i18n-tasks-use t('doc_auth.headings.document_capture_front')
-      # i18n-tasks-use t('doc_auth.headings.document_capture_back')
-      # i18n-tasks-use t('doc_auth.headings.document_capture_selfie')
-      I18n.t("doc_auth.headings.document_capture_#{attr}", options)
+    def validate_images
+      errors.add(:front, t('doc_auth.errors.not_a_file')) if front.is_a? URI::InvalidURIError
+      errors.add(:back, t('doc_auth.errors.not_a_file')) if back.is_a? URI::InvalidURIError
+      errors.add(:selfie, t('doc_auth.errors.not_a_file')) if selfie.is_a? URI::InvalidURIError
     end
 
-    private
-
-    attr_reader :params
-
     def throttle_if_rate_limited
       return unless @throttled
       errors.add(:limit, t('errors.doc_auth.acuant_throttle'))
     end
 
-    def throttled_else_increment
-      return unless document_capture_session
-      @throttled = Throttler::IsThrottledElseIncrement.call(
-        document_capture_session.user_id,
-        :idv_acuant,
-      )
+    def self.human_attribute_name(attr, options = {})
+      I18n.t("doc_auth.headings.document_capture_#{attr}", options)
```

Comment:
It appears it doesn't currently fail without the `i18n-tasks-use` comments, but those are meaningfully understood by `i18n-tasks` for its `unused` command, and at least I find it helpful as a developer to understand at a glance what variations we're expecting. Should we consider keeping those?

```suggestion
      # i18n-tasks-use t('doc_auth.headings.document_capture_front')
      # i18n-tasks-use t('doc_auth.headings.document_capture_back')
      # i18n-tasks-use t('doc_auth.headings.document_capture_selfie')
      I18n.t("doc_auth.headings.document_capture_#{attr}", options)
```